guides/source/upgrading_ruby_on_rails.md

```diff
@@ -169,81 +169,94 @@ more meaningful preview images. Previously the first frame of the video would be
 and that caused problems if the video faded in from black. This change requires
 FFmpeg v3.4+.
 
-### ActiveStorage variant processor changed to :vips
+### ActiveStorage default variant processor changed to `:vips`
 
-Image transformation will now use libvips instead of ImageMagick. This will reduce
+For new apps, image transformation will use libvips instead of ImageMagick. This will reduce
 the time taken to generate variants as well as CPU and memory usage, improving response
-times in apps that rely on active storage to serve their images. 
+times in apps that rely on active storage to serve their images.
 
 The `:mini_magick` option is not being deprecated, so it is fine to keep using it.
 
-Migrating to libvips requires changing existing image transformation code to the
-`image_processing` macros, and replacing ImageMagick's options with libvips' options.
+To migrate an existing app to libvips, set:
+
+```ruby
+Rails.application.config.active_storage.variant_processor = :vips
+```
+
+You will then need tochange existing image transformation code to the
+`image_processing` macros, and replace ImageMagick's options with libvips' options.
 
 #### Replace resize with resize_to_limit
 ```diff
 - variant(resize: "100x")
 + variant(resize_to_limit: [100, nil])
 ```
 
-If you forget to do this, when you switch to vips you will see this error: `no implicit conversion to float from string`.
+If you don't do this, when you switch to vips you will see this error: `no implicit conversion to float from string`.
```

Comment:
`forget to` seems better to read, lets retain that.

Reply:
I don't agree - I think it is prescribing a motivation that isn't relevant. Whether you forget, or you just don't bother, or you choose not to, you will get the same outcome. It's also more concise this way.

Reply:
> or you just don't bother, or you choose not to

When would someone choose not to? (Is that a valid case to assume?)

Reply:
🤷  I can't think of any good reasons. I acknowledge this is a pretty semantic argument.

Reply:
If you switch to vips your code will break if you don’t change. 

I assumed someone changing to vips who did not change this, would have forgotten about the change (like I did). That was my motivation for “forget”. 

That said, english is not my first language and I don’t speak it daily, so I’m not familiar with the fine points of the language. And as a novice in contributing, I’m not familiar with Rails language for documentation. 

However, as somone who speaks Brazilian Portuguese, “don’t do” sounds more neutral than “forget to do”.

Maybe we need a guide on language for Rails? At least for people contributing to documentation?

Reply:
Yeah, I don't think there's an official language guide, as long as it makes sense that's 99% of it. I just wrote this the way I would say it to someone in person, which I think is a good tone for this sort of guide.

Reply:
There is the wording chapter in the API guide: https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#wording

Reply:
I think moving the comma makes more sense, as "not doing this" only applies to moving to vips.
I also think having `this` refer to something later in the sentence is a bit odd.
```suggestion
If you don't do this when migrating to vips, you will see the following error: `no implicit conversion to float from string`.
```
Or maybe:
```suggestion
If you don't do this when migrating to vips, it will raise an error with `no implicit conversion to float from string`.
```